build-tools/packages/build-cli/src/machines/fluidReleaseMachine.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import path from "path";
+import { FileSystem as fs } from "@rushstack/node-core-library";
+import { from as createStateMachine } from "jssm";
+
+// eslint-disable-next-line unicorn/prefer-module
+const machineDefinitionFile = path.join(__dirname, "FluidRelease.fsl");
+const fsl = fs.readFile(machineDefinitionFile).toString();
+
+/**
+ * An FSL state machine that encodes the Fluid release process.
+ *
+ * @alpha
```

Comment:
Do we have any guidance for use of this tag? I wasn't aware of us using it anywhere previously.